### PR TITLE
Post request send input and output lang

### PR DESCRIPTION
**Pull request by swalsh15**
Added drop down bars to select input and output language. Send these in a post request to backend

Screen Shot 2020-06-25 at 12 27 01 PM

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -21,7 +27,7 @@ class App extends React.Component {
       headers: new Headers({
         'Content-Type': 'application/x-www-form-urlencoded',
       }),
-      body: "input="+ this.state.input
+      body: "input="+ this.state.input + "&in_lang=" + this.state.inputLanguage + "&out_lang" + this.state.outputLangauge
```
Comment (g-harel) [inline src/App.js]: Building up the string ourselves is error prone, could be easier to use https://stackoverflow.com/a/49701878.

File: src/App.js [hunk 2]
```diff
@@ -37,10 +43,55 @@ class App extends React.Component {
...
+            <em>None</em>
+          </MenuItem>
+          <MenuItem value={"js"}>JavaScript</MenuItem>
+          <MenuItem value={"py"}>Python</MenuItem>
+        </Select>
+        </FormControl>
+        
+       
+        <FormControl>
+        <InputLabel id="demo-controlled-open-select-label">Output</InputLabel>
```
Comment (g-harel) [inline src/App.js]: Is this id being used for something? It seems to be duplicated too.

File: src/App.js (hunk 1 above)
Comment (g-harel) [reply src/App.js]: Would also be nice to do some validation, for example only sending when both in and out languages are set.

File: src/App.js (hunk 2 above)
Comment (swalsh15) [reply src/App.js]: I pasted from a material ui example - I'll remove it

**Pull request merged**